js/schedule.js

```diff
@@ -56,20 +64,26 @@ createLunchBreakfastRow = function(item) {
 createTracksRow = function(item) {
     var rowContent = '';
 
-    rowContent += "<div class='row'>"+
+    rowContent += "<div class='row '>"+
                     "<div class='col-xs-12 col-sm-12 col-lg-3 time schedule-column'>"+
                         "<span class='center-align'>"+
                             item.Time+
                         "</span>"+
                     "</div>";
 
     for (var i = 0; i < item.Tracks.length; i++) {
-        rowContent += "<div class='col-xs-12 col-lg-2 schedule-column'>"+
-                            "<strong>"+
-                                "<a href='"+item.Tracks[i].Url+"' target='_blank'"+
-                                    "title='"+item.Tracks[i].Title+"'>"+
+        rowContent += "<div class='col-xs-12 col-lg-2 schedule-column '><strong><div class='popup text-center'>"+
+                            "<div class='row'><div class='col-2 col-sm-2 col-md-5 col-lg-5'>"+
```

Comment:
The grids are not properly used here. You need to use something like `col-xs-12 col-md-3` here and `col-xs-12 col-md-9` in the next grid. This will stack the grids for smaller screen widths. This will introduce a positioning issue for the popup which you need to fix with proper css.